test/integration/suites/nested-rotation/intermediateA/agent/agent.conf

```diff
@@ -0,0 +1,33 @@
+agent {
+    data_dir = "/opt/spire/data/agent"
+    log_level = "DEBUG"
+    server_address = "intermediateA-server"
+    server_port = "8081"
+    socket_path = "/opt/spire/sockets/workload_api.sock"
+    trust_bundle_path = "/opt/spire/conf/agent/bootstrap.crt"
+    trust_domain = "domain.test"
+}
+
+plugins {
+    NodeAttestor "x509pop" {
+        plugin_data {
+            private_key_path = "/opt/spire/conf/agent/agent.key.pem"
+            certificate_path = "/opt/spire/conf/agent/agent.crt.pem"
+        }
+    }
+    KeyManager "disk" {
+        plugin_data {
+            directory = "/opt/spire/data/agent"
+        }
+    }
+    WorkloadAttestor "unix" {
+        plugin_data {
+        }
+    }
+    WorkloadAttestor "docker" {
+        plugin_data {
+            # Error when using default (1.41)
```

Comment:
What was the error here? Is this a problem we need to address in SPIRE?

Reply:
```root-agent_1            | time="2020-04-14T23:19:03Z" level=error msg="Failed to collect all selectors for PID" error="workload attestor \"docker\" failed: rpc error: code = Unknown desc = Error response from daemon: client version 1.41 is too new. Maximum supported API version is 1.40" pid=27 subsystem_name=workload_api```